src/SourceGenerators/Uno.UI.SourceGenerators.netcore.Tests/XamlCodeGeneratorTests/TestOutput/When_ObservableProperty_AttributeDoesNotExists/MainPage_d6cd66944958ced0c513e0a04797b51d.cs

```diff
@@ -0,0 +1,265 @@
+			{
+				global::Uno.UI.FrameworkElementHelper.SetBaseUri(c3, __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d);
+				c3.CreationComplete();
+			}
+			))
+			;
+			
+			this
+			.GenericApply(((c4) => 
+			{
+				// Source 0\MainPage.xaml (Line 1:2)
+				
+				// WARNING Property c4.base does not exist on {http://schemas.microsoft.com/winfx/2006/xaml/presentation}Page, the namespace is http://www.w3.org/XML/1998/namespace. This error was considered irrelevant by the XamlFileGenerator
+			}
+			))
+			.GenericApply(((c5) => 
+			{
+				// Class TestRepro.MainPage
+				global::Uno.UI.FrameworkElementHelper.SetBaseUri(c5, __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d);
+				c5.CreationComplete();
+			}
+			))
+			;
+			OnInitializeCompleted();
+
+			Bindings = new MainPage_Bindings(this);
+			Loading += (s, e) => 
+			{
+				__that.Bindings.Update();
+				__that.Bindings.UpdateResources();
+			}
+			;
+		}
+		partial void OnInitializeCompleted();
+		private global::Windows.UI.Xaml.Markup.ComponentHolder _component_0_Holder  = new global::Windows.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Windows.UI.Xaml.Controls.TextBlock _component_0
+		{
+			get
+			{
+				return (global::Windows.UI.Xaml.Controls.TextBlock)_component_0_Holder.Instance;
+			}
+			set
+			{
+				_component_0_Holder.Instance = value;
+			}
+		}
+		private global::Windows.UI.Xaml.Markup.ComponentHolder _component_1_Holder  = new global::Windows.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Windows.UI.Xaml.Controls.TextBlock _component_1
+		{
+			get
+			{
+				return (global::Windows.UI.Xaml.Controls.TextBlock)_component_1_Holder.Instance;
+			}
+			set
+			{
+				_component_1_Holder.Instance = value;
+			}
+		}
+		private global::Windows.UI.Xaml.Markup.ComponentHolder _component_2_Holder  = new global::Windows.UI.Xaml.Markup.ComponentHolder(isWeak: true);
+		private global::Windows.UI.Xaml.Controls.TextBlock _component_2
+		{
+			get
+			{
+				return (global::Windows.UI.Xaml.Controls.TextBlock)_component_2_Holder.Instance;
+			}
+			set
+			{
+				_component_2_Holder.Instance = value;
+			}
+		}
+		private interface IMainPage_Bindings
+		{
+			void Initialize();
+			void Update();
+			void UpdateResources();
+			void StopTracking();
+		}
+		#pragma warning disable 0169 //  Suppress unused field warning in case Bindings is not used.
+		private IMainPage_Bindings Bindings;
+		#pragma warning restore 0169
+		[global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+		private class MainPage_Bindings : IMainPage_Bindings
+		{
+			#if UNO_HAS_UIELEMENT_IMPLICIT_PINNING
+			private global::System.WeakReference _ownerReference;
+			private global::TestRepro.MainPage Owner { get => (global::TestRepro.MainPage)_ownerReference?.Target; set => _ownerReference = new global::System.WeakReference(value); }
+			#else
+			private global::TestRepro.MainPage Owner { get; set; }
+			#endif
```

Comment:
@jeromelaban Something is wrong here.

`UNO_HAS_UIELEMENT_IMPLICIT_PINNING` is only defined internally (in Uno.UI solution).

In user apps, we will always be hitting the `#else` part of the code.

I don't have enough background here, but if usage of WeakReference is required on some platforms, maybe we should propagate `UNO_HAS_UIELEMENT_IMPLICIT_PINNING` to user apps?

if it's not required anymore, then we should cleanup the generated code.

Reply:
Original fix was in https://github.com/unoplatform/uno/pull/3936

and it was reverted (possibly unintentionally) https://github.com/unoplatform/uno/pull/3974/files#diff-70bcc603cf98904b7de6118e7f166581de45b6911245b3ac7ba397096c1923b0L29

Reply:
Let's discuss this separately: https://github.com/unoplatform/uno/issues/11976